docs/blog-documentation.md

```diff
@@ -61,7 +61,7 @@ For custom logo on image
 
 ```
 imagetag: "Other"
-customLogo: "./static/logos/gsoc.png"
```

Comment:
Also keep PNG, JPG looks bad with black background/without transparency

Reply:
Let's just revert all changes to  the `docs/` folder (except for https://github.com/MovingBlocks/ModuleSite/pull/152/files#diff-5dd7cfaa91a80e95302a9cb1d0e0aaf34e9f6291ab5250990614f47e59c54c97L20) as the PNGs are already super small.